CO-61-update-docs-for-co-58 (#68)
containeroo/cloudflare-operator: docs/content/upgrade_notes.md

# Upgrade Notes

## From v0.1.3 to v0.2.0

> Properties of the cloudflare-operator CRD `ip.containeroo.ch` has BREAKING changes.  

Reviewer: ```suggestion
Properties of the cloudflare-operator CRD `ips.cf.containeroo.ch` has BREAKING changes.  
```

> You have to delete all `IP` objects before updating and re-create it according the new specification:

Reviewer: ```suggestion
You have to delete all `IP` objects before updating and re-create them according to the new specification:
```

```bash
kubectl delete ip --all
```

> New `ip.containeroo.ch` CRD specification can be found [here](/docs/content/core_concept.md#IP).

Reviewer: ```suggestion
New `ips.cf.containeroo.ch` CRD specification can be found [here](/docs/content/core_concept.md#IP).
```

> If you have create a `IP` object to sync your external IPv4 address and not set `dynamicIPSources`, you have to delete your object and re-create a new `IP` object.

Reviewer: ```suggestion
If you created an `IP` object to sync your external IPv4 address and did not set `dynamicIPSources`, you have to delete that object and create a new `IP` object.
```

Example:

```bash

> kubectl apply -f - << EOF

Reviewer: ```suggestion
kubectl apply -f - << EOF
---
```
